### PR TITLE
Suggestions from Jos THIS PR HAS BEEN REPLACED  by PR #109

### PR DESCRIPTION
**Pull request by andped10**
Revert `solvent_surface_coverage` to `solvation`
- Solvation is the term used in the reflectometry community

Change data files (.ort)
- Metadata changed from `neutrons` to `neutron`
- Okay to make changes to the meta data to prevent warnings in examples / tests
- No changes to numerical data

Change final line in solvation.ipynb
- To clarify what is being done

### CODE REVIEW

File: EasyReflectometry/sample/assemblies/surfactant_layer.py [hunk 1]
```diff
@@ -80,13 +80,13 @@ def from_pars(
         tail_layer_molecular_formula: str,
         tail_layer_thickness: float,
         tail_layer_solvent: Material,
-        tail_layer_solvent_surface_coverage: float,
```
[the remark this answers is not in the document]
Comment (jfkcooper) [reply EasyReflectometry/sample/assemblies/surfactant_layer.py:83]: Though they can be solvated by different things, so `head_layer_solvent` and `tail_layer_solvent` are fine

File: EasyReflectometry/sample/elements/layers/layer_apm.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def __init__(
         :param molecular_formula: Formula for the molecule in the layer.
         :param thickness: Layer thickness in Angstrom.
         :param solvent: Solvent containing the molecule.
-        :param solvent_surface_coverage: Fraction of layer not covered by molecule.
+        :param solvation: Fraction of layer not covered by molecule.
```
Comment (jfkcooper) [inline EasyReflectometry/sample/elements/layers/layer_apm.py:106]: "Fraction of layer not covered by molecule." --> "Fraction of layer not covered by surfactant layer"

File: EasyReflectometry/sample/elements/layers/layer_apm.py [hunk 3]
```diff
@@ -246,17 +244,17 @@ def solvent(self, new_solvent: Material) -> None:
         self.material.solvent = new_solvent
 
     @property
-    def solvent_surface_coverage(self) -> Parameter:
+    def solvation(self) -> Parameter:
         """Get the fraction of layer not covered by molecules."""
```
Comment (jfkcooper) [inline EasyReflectometry/sample/elements/layers/layer_apm.py:248]: molecules --> surfactant

File: EasyReflectometry/sample/elements/layers/layer_apm.py [hunk 4]
```diff
@@ -246,17 +244,17 @@ def solvent(self, new_solvent: Material) -> None:
...
         return self.material.fraction
 
-    @solvent_surface_coverage.setter
-    def solvent_surface_coverage(self, solvent_surface_coverage: float) -> None:
+    @solvation.setter
+    def solvation(self, solvation: float) -> None:
         """Set the fraction of layer not covered by molecules.
```
Comment (jfkcooper) [inline EasyReflectometry/sample/elements/layers/layer_apm.py:253]: molecules --> surfactant

File: docs/sample/assemblies_library.rst [hunk 5]
```diff
@@ -85,22 +85,22 @@ The creation of a :py:class:`SurfactantLayer` object takes a large number argume
     tail_formula = 'C30D64'
     head_thickness = 10.0
     tail_thickness = 16.0
-    head_solvent_surface_coverage = 0.2
-    tail_solvent_surface_coverage = 0.0
+    head_solvation = 0.2
```
Comment (jfkcooper) [inline docs/sample/assemblies_library.rst:88]: This is not a good example, as there must be the same number of head and tail groups, so the solvation should be the same

File: docs/tutorials/multi_contrast.ipynb [hunk 6]
```diff
@@ -227,22 +187,22 @@
...
    "id": "74e19578-03d4-4295-b6e2-a65fb2508c78",
    "metadata": {},
    "outputs": [],
    "source": [
     "head_thickness = 12\n",
     "tail_thickness = 20\n",
-    "head_solvent_surface_coverage = 0.5\n",
-    "tail_solvent_surface_coverage = 0.0\n",
+    "head_solvation = 0.5\n",
+    "tail_solvation = 0.0\n",
```
Comment (jfkcooper) [inline docs/tutorials/multi_contrast.ipynb:198]: THis is not a good example as the head and tail groups have to be equal, so the solvation should be the same between them

**Review by jfkcooper — CHANGES_REQUESTED** [review]
Mostly fine, but the surfactant should be described by a single solvation parameter, rather than one for the head and another for the tail, since this doesn't make so much physical sense (to allow them to be different while forcing the number of head and tail groups to be the same).

File: EasyReflectometry/sample/assemblies/surfactant_layer.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arm61) [reply EasyReflectometry/sample/assemblies/surfactant_layer.py:85]: I think they can be solvated differently, i.e., a hydrophilic head will have more water (more solvation) than a hydrophobic tail no?

[the remark this answers is not in the document]
Comment (jfkcooper) [reply EasyReflectometry/sample/assemblies/surfactant_layer.py:85]: The head will have more water yes, but if we are talking about solvation in terms of surface coverage, then no (at least from my point of view). The head group SLD needs to be defined in terms of some fraction of water, and then the whole structure has some surface coverage, which I think is being called the salvation in this case.

[the remark this answers is not in the document]
Comment (jfkcooper) [reply EasyReflectometry/sample/assemblies/surfactant_layer.py:85]: After some discussion, we came to the conclusion that we are talking about slightly different things, and yes they can be solvated differently, but the coverage for both must be the same, so something like `head_solvation` and `tail_solvation` should exist, but so should `coverage`.

**Comment by andped10** [comment]
Was handled in #109

**Pull request closed without merging**